spec/rails_helper.rb

```diff
@@ -58,7 +58,7 @@
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  # config.use_transactional_fixtures = true
```

Comment:
It is being set to false in a different place already (spec/spec_helper.rb). It made more sense for me to do that there since the comment explaining why it's false is over there as well. Otherwise I would agree :)